serving/docker/dockerd-entrypoint-with-cuda-compat.sh

```diff
@@ -19,7 +19,7 @@ translateTGIToLMI() {
 if [ -f /usr/local/cuda/compat/libcuda.so.1 ]; then
     CUDA_COMPAT_MAX_DRIVER_VERSION=$(readlink /usr/local/cuda/compat/libcuda.so.1 |cut -d'.' -f 3-)
     echo "CUDA compat package requires Nvidia driver ⩽${CUDA_COMPAT_MAX_DRIVER_VERSION}"
-    NVIDIA_DRIVER_VERSION=$(sed -n 's/^NVRM.*Kernel Module *\([0-9.]*\).*$/\1/p' /proc/driver/nvidia/version 2>/dev/null || true)
+    NVIDIA_DRIVER_VERSION=$(nvidia-smi --query-gpu=driver_version --format=csv,noheader --id=0 2>/dev/null | cut -d'.' -f1 || true)
```

Comment:
what is this for?

Reply:
I tried the sed command, but it doesn't work, returns an empty string

Reply:
existing command didn't work for me, probably they changed the version file?

If we want to use the version file, this worked `grep -oP '(\d+\.\d+\.\d+)' /proc/driver/nvidia/version | sed -n '1p'`.
I prefer using `nvidia-smi` as that's more standard.